src/WebJobs.Script.Host/Program.cs

```diff
@@ -21,14 +25,27 @@ public static void Main(string[] args)
                 rootPath = (string)args[0];
             }
 
-            var config = new ScriptHostConfiguration()
-            {
-                RootScriptPath = rootPath,
-                IsSelfHost = true
-            };
+            var host = new HostBuilder()
+                .ConfigureLogging(b =>
+                {
+                    b.SetMinimumLevel(LogLevel.Information);
+                    b.AddConsole();
```

Comment:
Are we bringing back the console logger? Related issue: https://github.com/Azure/azure-functions-host/issues/2596

Reply:
This is in the sample app. Console logger will be available if configured, though.